src/components/NewCard/NewCard.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import styles from './NewCard.st.css';
+import { NEWCARD_DATA_HOOKS, NEWCARD_DATA_KEYS } from './dataHooks';
+
+export interface NewCardProps {
+  stacked?: boolean;
+  className?: string;
+  'data-hook'?: string;
+}
+
+export interface ContainerProps {
+  children: React.ReactNode;
+  minWidth?: number;
+  className?: string;
+  'data-hook'?: string;
+}
+
+interface DefaultProps {
+  stacked: false;
+  'data-hook': string;
+}
+
+const Container: React.FunctionComponent<ContainerProps> = props => {
+  const { className, children, minWidth } = props;
+  return (
+    <div
+      style={{ flexBasis: `${minWidth}px` }}
```

Comment:
🤔 I think we might need to add the minWidth conditionally, only if it exists.
wdyt?

Reply:
It won't be added if `minWidth = undefined`. 
I can add a condition so that logic will be explicit.